evm/test/BasicConsumer_test.js

```diff
@@ -4,14 +4,22 @@ import { assertBigNum } from '../src/matchers'
 const BasicConsumer = artifacts.require('BasicConsumer.sol')
 const Oracle = artifacts.require('Oracle.sol')
 
+let roles
+
+before(async () => {
```

Comment:
Could we cut down this boiler plate and call `h.initializeRolesAndPersonas()` in the module, and then export roles, personas...?

Reply:
its asynchronous initialization so you can't synchronously call it, hence the `before` hook

Reply:
except we don't want to use that hook within `helpers.ts` since that now introduces side-effects on import. it also assumes the importer is using `mocha` for their test framework, which may not be the case for external module consumers